data/sql/updates/pending_db_world/rev_1553372596303652700.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1553372596303652700');
+INSERT INTO `creature_template_addon` (`entry`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES
+(33691,0,0,0,0,0,'62019'); -- Rune of Summoning
```

Comment:
please always use `DELETE` before `INSERT` (even if this value is not present on a clean AC)

Reply:
I mean for the `creature_template_addon`, no need for the `version_db_world`

Reply:
Lol, i thought about it and i forgot to add it.